app/views/conferences/_conference_details.html.haml

```diff
@@ -40,3 +40,7 @@
               = link_to 'Unsubscribe', conference_subscriptions_path(conference.short_title), method: :delete, class: 'btn btn-default'
             - if current_user && current_user.physical_tickets.by_conference(conference).any?
               = link_to "My Tickets", conference_physical_ticket_index_path(conference.short_title), class: 'btn btn-default'
+            - if current_user && current_user.booths.where(conference_id: conference.id).count > 0
+              = link_to 'My Booth Requests', conference_booths_path(conference.short_title), class: 'btn btn-default'
+            - elsif can? :new, conference.booths.new
+              = link_to 'Request Booth', new_conference_booth_path(conference.short_title)
```

Comment:
Putting this at the very end might be a little out of place. What about moving it after 'My proposals' ?